### ISSUE TITLE
Unnecessary comma in sckenhancedselection_content_field.html.twig

### ISSUE THREAD

**Issue by kossuth-wt** [netgen/NetgenEnhancedSelectionBundle#28]
Bundle version is 3.3.0

In sckenhancedselection_content_field.html.twig line 16 there might be an unnecessary comma displayed.

Steps to reproduce:
- Have a content type with this field type with 3 options
- Create a content and select any of the options except for the last one

The for cycle goes through the available options, and if not the last is selected the "if not loop.last" will be true and displays a delimiter, even if this was the last selected option to display.

**Comment by emodric** [comment]
@kossuth-wt Hi!

Can you test the following version of the template which would fix your problem?

```twig
{% block sckenhancedselection_field %}
{% spaceless %}

{% set available_options = fieldSettings.options %}
{% set identifiers = field.value.identifiers %}

{% if fieldSettings.delimiter is not empty %}
    {% set delimiter = fieldSettings.delimiter %}
{% else %}
    {% set delimiter = ', ' %}
{% endif %}

{% if available_options is not empty and identifiers is not empty %}
    {% set options = [] %}

    {% for option in available_options %}
        {% if option.identifier in identifiers %}
            {% set options = options|merge([option.name]) %}
        {% endif %}
    {% endfor %}

    {{ options|join(delimiter) }}
{% elseif identifiers is not empty %}
    {{ identifiers|join(delimiter) }}
{% endif %}

{% endspaceless %}
{% endblock %}
```

**Comment by emodric** [comment]
@kossuth-wt confirmed on eZ slack that the template works. Fixed in 3.3.1.
